libstuff/libstuff.cpp

```diff
@@ -1808,6 +1808,9 @@ int S_socket(const string& host, bool isTCP, bool isPort, bool isBlocking) {
             if (setsockopt(s, SOL_SOCKET, SO_REUSEADDR, (char*)&enable, sizeof(enable)))
                 STHROW("couldn't set REUSEADDR");
 
+            if (setsockopt(s, SOL_SOCKET, SO_REUSEPORT, (char*)&enable, sizeof(enable)))
```

Comment:
Can we add a comment here explaining what/why this does what it does, and update the one above?

Reply:
added